packages/perseus/src/widgets/interactive-graphs/types.ts

```diff
@@ -71,6 +71,7 @@ export interface CircleGraphState extends InteractiveGraphStateCommon {
     type: "circle";
     center: Coord;
     radius: number;
+    radiusPoint: Coord;
```

Comment:
Originally I was just using `radiusPoint`. I added back `radius` when I looked at grading, but I think I could remove it again.

Reply:
Okay, I remember what it was @benchristel: I wanted to store `radiusPoint` because I figured `radius` could just be computed from that and it'd give me some flexibility around positioning the handle.

However the way things worked was that `MafsGraph` was just blindingly passing its state up to `InteractiveGraph`. `InteractiveGraph` merges the object from the `onChange` callback into its state and without `radius` the editor doesn't work as expected.

This made me wonder: do we _have_ to be bound to the structure of state in the legacy interactive graph? So I updated my code to include `mafStateToInteractiveGraph`, the idea being that we could store whatever we want, however we want in the new code as long as we transform it to be compatible with the legacy code. I expect this could be controversial, but I also imagine it could be useful as we implement the weirder charts and try to add a11y improvements.

Obviously I could also just add `radius` back to the store.

Reply:
FWIW, I like the approach of keeping only the state we need in our `useReducer` store, and transforming it back to the legacy format only when we have to.

Currently, someone who's only looking at the useReducer state type has to wonder "why did they design it this way" and it's unsatisfying if the answer is "an accident of history."